### PR TITLE
[CELEBORN-1559]Fix make distribution script failed to recognize specific profile

### PR DESCRIPTION
**Pull request by zhaohehuhu**
### What changes were proposed in this pull request?
as title

### Why are the changes needed?
When I run the command(./build/make-distribution.sh -Pflink-1.16),  the profiles can be resolved and built.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Yes

### CODE REVIEW

**Comment by zhaohehuhu** [comment]
plz review this PR. Thanks @FMX

**Comment by SteNicholas** [comment]
@zhaohehuhu, please update the title of this pull request to link with ticket.

**Comment by zhaohehuhu** [comment]
> @zhaohehuhu, please update the title of this pull request to link with ticket.

Got it. Thanks.

File: build/make-distribution.sh [hunk 1]
```diff
@@ -262,7 +256,8 @@ function sbt_build_service {
 
   echo "Celeborn $VERSION$GITREVSTRING" > "$DIST_DIR/RELEASE"
   echo "Build flags: $@" >> "$DIST_DIR/RELEASE"
-  if [[ "$HADOOP_AWS_ENABLED" == "true" ]]; then
+
+  if [[ $@ == *"hadoop-aws"* ]]; then
     export SBT_MAVEN_PROFILES="hadoop-aws"
```
Comment (FMX) [inline build/make-distribution.sh:261]: There is no need to export this variable. Just setting this variable should be enough.

Comment (zhaohehuhu) [reply build/make-distribution.sh]: Done. Thanks

**Review by FMX — APPROVED** [review]
LGTM. Thanks. Merged into main(v0.6.0).

**Pull request closed without merging**